astropy/time/tests/test_basic.py

```diff
@@ -1076,6 +1079,9 @@ def test_to_datetime_pytz():
         assert tz.tzname(dt) == tz_dt.tzname()
     assert np.all(time == forced_to_astropy_time)
 
+    with pytest.raises(ValueError):
```

Comment:
I don't think we need this second test, or does it really do something different?